docs/modules.rst

```diff
@@ -177,19 +177,40 @@ Demonstration of module action blocks:
 Actions
 =======
 
-Actions are tasks for Astrality to perform, and are placed within :ref:`action blocks <modules_action_blocks>` in order to specify *when* to perform them. There are four available ``action`` types:
+Actions are tasks for Astrality to perform, and are placed within :ref:`action
+blocks <modules_action_blocks>` in order to specify *when* to perform them.
+These are the available ``action`` types:
 
     :ref:`import_context <context_import_action>`:
-        Import a ``context`` section from a YAML formatted file. ``context`` variables are used as replacement values for placeholders in your :ref:`templates <templating>`. See :ref:`context <context>` for more information.
+        Import a ``context`` section from a YAML formatted file. ``context``
+        variables are used as replacement values for placeholders in your
+        :ref:`templates <templating>`. See :ref:`context <context>` for more
+        information.
 
     :ref:`compile <compile_action>`:
-        Compile a specific template to a target path.
+        Compile a specific template or template directory to a target path.
+
+    :ref:`copy <copy_action>`:
+        Copy a specific file or directory to a target path.
+
+    :ref:`symlink <symlink_action>`:
+        Create symbolic link(s) pointing to a specific file or directory.
+
+    :ref:`stow <stow_action>`:
```

Comment:
I think this explanation is a little confusing from an outsider's perspectives. Maybe say something like:

> Process a whole directory treating contents as either templates or non-templates based on a pattern. Non-templates can either be ignored, copied or symlinked. Directory and file hierarchy is maintained from the `content` to the `target` path.

IDK, still seems confusing. Needs clarification

Reply:
Oops, did not see the detailed section below. No changes needed

Reply:
Yes, it is just a short overview of all the available actions before going more in depth ;)

I'm at any moment finished rewriting all the new Action types, I just need to instantiate them in the ``Module`` class, and execute them in the appropiate order. I also need to ensure that files that have been copied will be re-copied when they are modified (just as with modified templates). The only issue is that I'm not sure if I should rename the global config option ``recompile_modified_templates`` as it also includes copied files after merging this PR...

Reply:
For static files, you could potentially avoid copying if the source/dest files' timestamps are the same. Either way, the global config option should probably be renamed. Maybe something like `reprocess_modified_files`. Here the `modified` part also implies doing something (such as copying) only if necessary

Reply:
Okay, I will rename the option to ``reprocess_modified_files`` then!